### ISSUE TITLE
Structures showing up in Twilight Forest (dimension whitelist request)

### ISSUE THREAD

**Issue by laserlemons** [TelepathicGrunt/RepurposedStructures#88]
In the config there is only the option to blacklist dimensions. As time goes on this is going to get tedious for pack developers to keep track of all of the dimensions in their pack and remember to add them all to this mod's config. Please implement a whitelist instead.

**Comment by TelepathicGrunt** [comment]
i can see about white list in the future but not now. Also, if you delete the config, it will regenerate with all the dimensions i blacklist by default currentl. The default string is this right now which does include twilightforest:
blacklistedDimensions = "the_bumblezone:the_bumblezone, twilightforest:twilightforest, undergarden:undergarden, the_midnight:the_midnight, theabyss:theabyssdim, theabyss:theabyssiceworld, theabyss:death, theabyss:the_end_of_time, theabyss:the_end_of_time_2, theabyss:dream, theabyss:dream_2, theabyss:dream_3, theabyss:radio, theabyss:theabyssdimgroundlands, theabyss:theabyssdimskylands, lostcities:lostcity"

**Comment by TelepathicGrunt** [comment]
Currently waiting on this PR to be merged before any further changes to dimension configs as this fixes an important bug with the dimension whitelist/blacklist system being used. https://github.com/MinecraftForge/MinecraftForge/pull/7777

**Comment by TelepathicGrunt** [comment]
Changed plans! 1.16.5-3.0.0 repurposed structures now has a config file dedicated to dimension whitelisting/blacklisting now! it is in the repurpsoed_structures-forge folder in the config folder. You can do `"all": "*"` in disallowedDimensions  to blacklist all dimensions and then pick and choose which dimension you want to spawn structures in with the allowedDimensions config entry. allowedDimensions overrides disallowedDimensions
